saltproc/abc.py

```diff
@@ -99,21 +102,37 @@ def read_depleted_materials(self, read_at_end=False):
                 :class:`Materialflow` object holding material composition and properties.
 
         """
-
-    @abstractmethod
-    def run_depletion_step(self, mpi_args, threads):
+    def run_depletion_step(self, mpi_args, args):
         """Runs a depletion step as a subprocess with the given parameters.
 
         Parameters
         ----------
         mpi_args : list of str
             Arguments for running simulations on supercomputers using
             ``mpiexec`` or similar programs.
-        threads : int
+        args : int
```

Comment:
Since you changed the name of threads to args, shouldn't the docstring also change? Otherwise, I'm not sure why this needed to be args rather than threads.